mne_bids_pipeline/_config.py

```diff
@@ -95,6 +94,11 @@
 The task to process.
 """
 
+task_is_rest: bool = False
+"""
+Whether the task should be treated as resting-state data.
+"""
```

Comment:
All changes like this are just copy-paste moves to match orders in `.md` files above, no content changes.